src/scripts/extensions/authenticationHelper.ts

```diff
@@ -154,7 +160,13 @@ export class AuthenticationHelper {
 				// The false case is expected behavior if the user has not signed in or credentials have expired
 				resolve({ parsedResponse: this.isValidUserInformationJsonString(response) ? response : undefined, request: request });
 			}, (error: OneNoteApi.RequestError) => {
+				retrieveUserInformationEvent.setStatus(Log.Status.Failed);
```

Comment:
Added https://github.com/OneNoteDev/WebClipper/issues/98 to track the use of OneNoteApi.RequestError in places where it doesn't make sense.